README.md

```diff
@@ -1,2 +1,36 @@
-# Tic-Tac-Toe-OOP
-OOP (Object Oriented Programming) tic tac toe project
+# Tic Tac Toe OOP (Object Oriented Programming)
+
+- Original project specification at [The Odin Project](https://www.theodinproject.com/courses/ruby-programming/lessons/oop)
+- Tic-tac-toe [From Wikipedia](https://en.wikipedia.org/wiki/Tic-tac-toe)
+
+<p align="center">
+  <img src="img/3b2hm4.gif">
+</p>
+
+## Game Rule:
+
+Tic Tac Toe is a Cool Math Games:
+Xs and Os is a paper-and-pencil game for two players, X and O:
+
+- Players take turns marking the spaces in a 3×3 grid.
+- The player who succeeds in placing three of their marks in a horizontal, vertical, or diagonal row wins the game.
+
+The following example game is won by the first player, X:
+![example](https://upload.wikimedia.org/wikipedia/commons/thumb/1/1b/Tic-tac-toe-game-1.svg/1280px-Tic-tac-toe-game-1.svg.png)
+
+## Technologies:
+
+- Ruby
+
+## Instructions:
+
+To play Tic-tac-toe:
+
+- clone or download this project [from github](https://github.com/Anna-Myzukina/Tic-Tac-Toe-OOP)
+- cd into the /bin directory
+- run the main.rb file,using command: ruby main.rb
```

Comment:
Here too :smile: 
```
ruby main.rb
```

Also, the advantage of this (`#!/usr/bin/env ruby`) is so we can run the game without the ruby` keyword. So running this (:point_down:) will work too. :blush: 

```
main.rb
```

Reply:
Thank you. Fixed them all.